microsoft-edge/webview2/release-notes/index.md

```diff
@@ -129,6 +129,12 @@ Release Date: November 18, 2024
 For full API compatibility, this Release version of the WebView2 SDK requires WebView2 Runtime version 131.0.2903.40 or higher.
 
 
+<!-- ------------------------------ -->
+#### General features
```

Comment:
use a "General features" heading at top of h2 section like 7x in Archive, or "Other changes" heading below Bug fixes section?

RelNotes > **General features**
https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3326#general-features

RelNotes > **Other changes**
https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3326#other-changes

Archive > **General features** - 7 examples:
https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/archive?branch=pr-en-us-3326#general-features --  Find "General features" 7x.  It's near top of h2 section, 7x.  0 hits on "Other changes" heading.

@victorhuangwq @sivMSFT

Reply:
General features feels weird for me as this isn't a new feature, but it's a change

Reply:
result:
**General changes** subhead at end of h2 section